gapis/gfxapi/test/gfxapi_test_import/imported.go

```diff
@@ -1,49 +0,0 @@
-// Copyright (C) 2017 Google Inc.
```

Comment:
This file seems pretty empty, so assume it was dropped on purpose during the move?